FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -904,61 +1324,54 @@ components:
         - code
         - severity
 
-    FaultRegistrationInformation:
-      type: object
-      properties:
-        uuid:
-          type: string
-          format: uuid
-        name:
-          type: string
-          description: The name or summary of the fault
-        source:
-          type: string
-          format: uuid
-        code:
-          type: integer
-          description: The code of the fault. Usually represented as an integer
-        timeOfDetection:
-          type: string
-          format: date-time
-          description: The time that the fault was first reported
-        faultTimeout: 
-          type: integer
-          minimum: 1
-          example: 100
-          description: Seconds within which a fault update is expected from the client. If no update is received, the fault is assumed to be resolved.
-          
+    ActiveComponent:
+      description: The final component data that are used for external consumption by displays and other services. These data may be altered between initial receipt and this value. 
+      allOf:
+        - $ref: '#/components/schemas/MonitoredComponent'
+        - type: object
+          properties:
+            selfReportedState:
+              description: The original self-reported state of the component. Useful for when an override is performed due to a fault or other special circumstance. 
+              $ref: '#/components/schemas/State'
+              
+            performanceStatistics:
+              $ref: '#/components/schemas/ComponentStatistics'
+
+    ActiveFault:
+      description: The final 'fault' that is used for external consumption by displays and other services
```

Comment:
This is good, but I think a contrasting statement between Fault and ActiveFault would add a lot of value.

Reply:
Addressed